src/server/req_runjob.c

```diff
@@ -545,6 +546,10 @@ req_runjob(struct batch_request *preq)
 		if (pjob) {
 			/* free prov_vnode before use */
 			job_attr_def[(int) JOB_ATR_prov_vnode].at_free(&pjob->ji_wattr[(int) JOB_ATR_prov_vnode]);
+			ptbl = parent->ji_ajtrk;
+			if (ptbl == NULL)
+				return;
+			ptbl->tkm_flags |= TKMFLG_RUN_JOB_REQ;
```

Comment:
I think you need to also do this for the range of subjobs code below

Reply:
Please refer to the line what you are talking about?

Reply:
Ignoring the range case, since it is not a supported feature. I didn't see any occurrences in the 40k Job array run.

Reply:
I believe running a range of subjobs is a supported feature.  The scheduler doesn't run a range because it is hard for it.  It needs to provide a destination, so it would only be able to run a range if they were all going to the same place.  Running a range is mainly used for qrun.